frontend/web/components/pages/OrganisationSettingsPage.js

```diff
@@ -237,6 +242,9 @@ const OrganisationSettingsPage = class extends Component {
                       ...[
                         SettingsTab.General,
                         paymentsEnabled && !isAWS ? SettingsTab.Billing : null,
+                        isEnterprise && isSaas === false
```

Comment:
Minor but cant this just be isEnterprise? I'd have thought that would make saas always false